documentation/pwa/tutorial-offline-form-submission.asciidoc

```diff
@@ -0,0 +1,26 @@
+---
+title: Offline Form Submission
+order: 8
+layout: page
+---
+
+= Offline Form Submission
+
+Our TypeScript <<../client-side-forms/tutorial-binder#,Binder>> works seamlessly with the <<tutorial-defer-endpoint-request#, deferrable endpoints>>, which allows your form submission to work both online and offline. 
+
+When the application is online, the form will be submitted normally. When offline, the form will be cached locally in the browser and be submitted automatically when coming back online.
```

Comment:
“Caching” typically means additional storage, while here it is the only one storage for the offline period of time. Let us rather say ”stored”.

```suggestion
When the application is online, the form is submitted normally. When offline, the form is stored locally in the browser and is submitted automatically when coming back online.
```

Reply:
done